workflow/kfdrc_RNAseq_workflow.cwl

```diff
@@ -872,5 +872,5 @@ hints:
 - SE
 - STAR
 "sbg:links":
-- id: 'https://github.com/kids-first/kf-rnaseq-workflow/releases/tag/v4.3.0'
+- id: 'https://github.com/kids-first/kf-rnaseq-workflow/releases/tag/v4.3.1'
```

Comment:
Is this really a bug fix? Seems more substantial.

Reply:
yeah, I was debating that a bit...probably enough to justify a 4.4.0

Reply:
Like, under the hood a lot happened, from a user perspective, aside from the tools, their interaction with the wf is the same

Reply:
Well that sounds like a minor update: `MINOR version when you add functionality in a backwards compatible manner`. Going to `4.4.0` is appropriate.